java/src/jmri/jmrit/display/layoutEditor/LayoutTrack.java

```diff
@@ -217,5 +211,80 @@ public Rectangle2D getBounds() {
         return new Rectangle2D.Double();
     }
 
+    /**
+     * get the object connected to this track for this connection type
+     * @param connectionType
+     * @return the object connected to this layout track connected for the connection type
+     * @throws jmri.JmriException
+     */
+    public Object getConnection(int connectionType) /* throws jmri.JmriException */ {
```

Comment:
Iit calls the LayoutTrack subclass getConnection methods that do throw this exception.